62.md

```diff
@@ -0,0 +1,103 @@
+NIP-62
+======
+
+Collections
+----------
+
+`draft` `optional`
+
+This NIP defines the minimum specification for collections - ordered, optionally-hierarchical assemblies of nostr events. Collections provide a standard way to organize and present related content, similar to how books organize chapters or journals organize articles.
+
+## Event Kinds
+
+This NIP defines two primary event kinds:
+
+- `30040`: Collection Index
+- `30041`: Collection Section
+
+## kind 30040: Collection Index
+
+A collection index defines the structure and metadata of a collection. It serves as a table of contents that references the actual content sections.
+
+### Requirements
+
+- The `content` field MUST be empty
+- MUST include a `title` tag containing the full title of the collection
+- MUST be uniquely identifiable by the combination of:
+  - `d` tag
+  - `pubkey`
+- MUST include `e` tags listing the events in desired display order
+  - Referenced events SHOULD be kind 30041 sections or nested kind 30040 indices
+  - Additional event kinds MAY be supported
+- For derivative works:
+  - MUST include a `p` tag identifying the original author
+  - MUST include an `E` tag referencing the original event immediately after the `p` tag
+
+### Example Index
+
+```json
+{
+    "id": "<event_id>",
+    "pubkey": "<event_originator_pubkey>",
+    "created_at": 1725087283,
+    "kind": 30040,
+    "tags": [
+        ["d", "aesop's-fables-by-aesop"],
+        ["title", "Aesop's Fables"],
+        ["author", "Aesop"],
+        ["i", "isbn:9780765382030"],
+        ["t", "fables", "classical", "literature"],
```

Comment:
```
["t", "fables"],
["t", "classical"],
["t", "literature"],
```
might be more useful, as then one could query for either of the tags. It also matches the current definition of the `t` tag in README.md.

Reply:
Ah, you're right, just checked and it's like that in my last Habla-generated article.